catalog/nsili/catalog-nsili-common/src/test/java/org/codice/alliance/nsili/common/TestResultDAGConverter.java

```diff
@@ -130,6 +131,93 @@ public void testMandatoryAttributesSuccess() throws Exception {
         assertThat(dag, notNullValue());
     }
 
+    @Test
+    public void testCreateAttribute() throws Exception {
+        String id = UUID.randomUUID()
+                .toString();
+        MetacardImpl card = new MetacardImpl();
+        card.setId(id);
+        card.setTitle("Test Title");
+        card.setSourceId("Test Source");
+        Date testDate = new Date(1000);
+        card.setCreatedDate(testDate);
+        card.setModifiedDate(testDate);
+
+        ResultImpl result = new ResultImpl();
+        result.setMetacard(card);
+
+        DAG dag = ResultDAGConverter.convertResult(result,
+                orb,
+                rootPOA,
+                new ArrayList<>(),
+                new HashMap<>());
+        assertThat(dag, notNullValue());
+
+        String attributeName = NsiliConstants.NSIL_PRODUCT + ":" + NsiliConstants.NSIL_CARD + "." + NsiliConstants.STATUS;
+
+        String value = ResultDAGConverter.getAttributeMap(dag).get(attributeName);
+        assertThat(value, is(NsiliCardStatus.NEW.name()));
+    }
+
+    @Test
+    public void testChangeAttribute() throws Exception {
+        String id = UUID.randomUUID()
+                .toString();
+        MetacardImpl card = new MetacardImpl();
+        card.setId(id);
+        card.setTitle("Test Title");
```

Comment:
Modified this to put the common code in a single function. Also added some statics for title and source.